06_correlation_analysis.ipynb

```diff
@@ -0,0 +1,1984 @@
+{
```

Comment:
<p>Each point on the plot on the right is a cancer type, right? I would add some description of what you see here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/pancancer-evaluation/pull/46/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Yep! I'll add a description.</p>